nixos/modules/services/monitoring/librenms.nix

```diff
@@ -43,6 +43,15 @@ let
     $sudo ${package}/artisan "$@"
   '';
 
+  libreNMSSyslogWrapper = pkgs.writeShellScriptBin "librenms-syslog" ''
+    cd ${pkgs.librenms}
+    sudo=exec
+    if [[ "$USER" != ${cfg.user} ]]; then
+      sudo='exec /run/wrappers/bin/sudo -u ${cfg.user}'
+    fi
+    $sudo ${package.phpPackage}/bin/php ${package}/syslog.php "$@"
```

Comment:
the sudo step here seems a bit weird, not sure why the binary is not just run as the required user?